real_code/hello.js

```diff
@@ -0,0 +1,35 @@
+$(document).ready(function(){
+  var userChoice = prompt("Do you choose rock, paper or scissors?");
+  var computerChoice = Math.random();
+
+
+
+if (computerChoice < 0.34) {
+	computerChoice = "rock";
+} else if(computerChoice <= 0.67) {
+	computerChoice = "paper";
+} else {
+	computerChoice = "scissors";
+} console.log("Computer: " + computerChoice);
+var compare = function(choice1,choice2) {
+    if(userChoice === computerChoice) {
+        return "The result is a tie!";
+    };
+    if(choice2 === "scissors") {
+    return "scissors wins";
+      $('body').append("scissors wins");
+}
+else {
+    return "paper wins";
+  $('body').append("paper wins");
+}
+if(choice2 === "scissors") {
+    return "rock wins";
+  $('body').append("rock wins");
+}
+else {
+    return "scissors wins";
+  $("body").append("scissors wins");
+}
+};
+});
```

Comment:
overall this is definitely heading in the right direction :tada:. just make sure you keep your code neatly formatted to make it easier to read. For example, make sure to use indentation when appropriate (inside code blocks), and try to maintain an equal width of indentation (use the tab key on your keyboard instead of the spacebar)